JumpscaleCore/clients/gedis/GedisClient.py

```diff
@@ -164,34 +164,22 @@ def _redis(self):
             # self._redis_.execute_command("config_format", "json")
             # authenticate us
             seed = j.data.idgenerator.generateGUID()  # any seed works, the more random the more secure
-            signature = self._nacl.default.sign_hex(seed)  # this is just std signing on nacl and hexifly it
+            signature = self._nacl.sign_hex(seed)  # this is just std signing on nacl and hexifly it
             self._redis_.execute_command("auth", self._threebot_me.tid, seed, signature)
 
         return self._redis_
 
-    # def __getattr__(self, name):
-    #     if name.startswith("_") or name in self._methods_gedis() or name in self._properties():
-    #         return self.__getattribute__(name)
-    #     return self.cmds.__getattribute__(name)
-
     @property
     def _threebot_me(self):
         if not self._threebot_me_:
-            if self.threebot_local_profile == "default":
-                self._threebot_me_ = j.tools.threebot.me.default
-            else:
-                print("TODO: implement")
-                j.shell()
-        j.shell()
+            # @TODO: shall we make a test here?!
+            # assert j.tools.threebot.me.exists(self.threebot_local_profile)
+            self._threebot_me_ = j.tools.threebot.me.get(self.threebot_local_profile)
```

Comment:
What will happen if the instance doesn't exist? It will just return a new instance with default values? Should you add `needexist=True` ?

Reply:
added `needexist=True` didn't know